lib/ama_layout/navigation.yml

```diff
@@ -81,26 +81,78 @@ non-member:
     link: "<%= Rails.configuration.gatekeeper_site %>/user/edit"
     icon: "fa-cogs"
 member-in-renewal:
-  - text: "Account Dashboard"
+  - text: "My Account Overview"
     link: "<%= Rails.configuration.gatekeeper_site %>/"
     icon: "fa-tachometer"
-  - text: "Renew"
-    link: "<%= Rails.configuration.youraccount_site %>/renew"
+  - text: "My Membership"
+    link: "<%= Rails.configuration.membership_site %>/membership/overview"
     icon: "fa-credit-card"
-  - text: "Driver Education"
+    sub_nav:
+      - text: "Membership Overview"
+        link: "<%= Rails.configuration.membership_site %>/membership/overview"
+      - text: "Renew Membership"
+        link: "<%= Rails.configuration.membership_site %>/renews/new"
+        icon: "fa-credit-card"
+      - text: "Manage Membership"
+        link: "<%= Rails.configuration.membership_site %>/membership/manage"
+      - text: "Billing Options"
+        link: "<%= Rails.configuration.youraccount_site %>/billing"
+      - text: "Print/Request New Card"
+        link: "<%= Rails.configuration.youraccount_site %>/membership"
+  - text: "My Roadside Assistance"
+    link: "<%= Rails.configuration.automotive_site %>/"
+    icon: "fa-car"
+    sub_nav:
+      - text: "Roadside Assistance Overview"
+        link: "<%= Rails.configuration.automotive_site %>/"
+  - text: "My Driver Education"
     link: "<%= Rails.configuration.driveredonline_site %>/"
     icon: "fa-car"
     sub_nav:
       - text: "Driver Education Overview"
         link: "<%= Rails.configuration.driveredonline_site %>/"
       - text: "New Driver Online Program"
         link: "<%= Rails.configuration.driveredonline_site %>/dashboard"
+  - text: "My AMA Rewards"
```

Comment:
can you add test for these new links and subnavs here https://github.com/amaabca/ama_layout/blob/e3793d9af45b27ec61a9423a5846c210b6073c96/spec/ama_layout/navigation_spec.rb